qcodes/instrument_drivers/ZI/ZIHDAWG8.py

```diff
@@ -19,6 +33,13 @@ class ZIHDAWG8(Instrument):
     (tested using LabOne 18.05.54618 and firmware 53866).
     Furthermore, the Data Server and Web Server must be running and a connection
     between the two must be made.
+
+    Compiler warnings, when uploading and compiling a sequence program, can
+    be treated as errors. This is desirable if the user of the driver does
+    not want clipping or truncating of waveform to happen silently by the
+    compiler. Warnings are constants on the module level and can be added to the
+    drivers attribute warnings_as_errors. If warning are added, they will raise
```

Comment:
Perfect! This is just the syntax for sphinx - a popular tool for generating documentation. Also, this way 
warnings_as_errors is not glued into the other text and "stands out" :)
```suggestion
    drivers attribute :attr:`warnings_as_errors`. If warning are added, they will raise
```

Reply:
sphinx is not happy about this:
`sphinx.errors.SphinxWarning: /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/qcodes/instrument_drivers/ZI/ZIHDAWG8.py:docstring of qcodes.instrument_drivers.ZI.ZIHDAWG8.ZIHDAWG8:7:py:attr reference target not found: warnings_as_error`

Reply:
weird. i guess your work around is fine. (i'm still learning myself how sphinx works and how one should write correct references to attributes and such)

Reply:
It did not work

Reply:
sad.

then wrapping it at least into double quote thingies like below should be nice:
```suggestion
    drivers attribute ``warnings_as_errors``. If warning are added, they will raise
```